paasta_tools/cli/cmds/spark_run.py

```diff
@@ -537,7 +579,14 @@ def get_spark_app_name(original_docker_cmd: Union[Any, str, List[str]]) -> str:
     if spark_app_name is None:
         spark_app_name = "paasta_spark_run"
 
-    spark_app_name += f"_{get_username()}"
+    spark_app_name += "_{}".format(get_username())
```

Comment:
In general spark_app_name should already be set and it should already have the username, at least for user-run batches.